pkg/operator/ceph/object/objectstore.go

```diff
@@ -632,8 +633,8 @@ func getObjectStores(context *Context) ([]string, error) {
 	return r.Realms, nil
 }
 
-func deletePools(ctx *Context, spec cephv1.ObjectStoreSpec, lastStore bool) error {
-	if emptyPool(spec.DataPool) && emptyPool(spec.MetadataPool) {
+func DeletePools(ctx *Context, metadataPool, dataPool cephv1.PoolSpec, lastStore bool, poolPrefix string) error {
+	if emptyPool(dataPool) && emptyPool(metadataPool) {
 		logger.Info("skipping removal of pools since not specified in the object store")
```

Comment:
This log line is out of date now that the implementation is reusable. I can't think of a way to still include it here, but since this is an info message we should keep it. Move this message to places where `DeletePools()` is called.